### PR TITLE
Win 10 installation notes

### PR DESCRIPTION
**Pull request by kerautret**
Using VS 2022

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ To build DGtal related examples, you would need:
   - [boost](http://boost.org) (>= 1.50).
   - the zlib package (already installed in many linux/macos distributions)
 
+For windows users, you can follows this [installations](https://github.com/DGtal-team/DGtal-Tutorials-DGMM2022/blob/main/windowsDGtalInstall.md) steps.
```
Comment (dcoeurjo) [inline README.md]:
```suggestion
For windows users, you can follow these [installation](https://github.com/DGtal-team/DGtal-Tutorials-DGMM2022/blob/main/windowsDGtalInstall.md) steps.
```

File: windowsDGtalInstall.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Installation of Dgtal on Windows
+These main steps are based on Visual Studio 2022, and the overall demarch is to generate first a **.sln** project from *cmake* command line and then open and build the project unde VS 2022. These steps were tested on Windows 10.
```
Comment (dcoeurjo) [inline windowsDGtalInstall.md]:
```suggestion
These main steps are based on Visual Studio 2022, and the overall workflow is to generate first a **.sln** project from *cmake* command line and then open and build the project under VS 2022. These steps were tested on Windows 10.
```

**Review by dcoeurjo — APPROVED** [review]
Thx, just minor stuff

**Comment by kerautret** [comment]
done thanks for the typos!

**Pull request merged**